### PR TITLE
add LICENSE

### PR DESCRIPTION
**Pull request by dhairyagandhi96**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+The code in this repository - the C wrapper and the parts in Julia - are
+licensed under the MIT license. The PyTorch library installed as a consequence
+of installing this library, via the Torch_jll artifact is separately licensed
+by PyTorch. The source code can be found at
+https://github.com/pytorch/pytorch/blob/master/LICENSE and its license can be
```
Comment (aviks) [inline LICENSE:27]: This is a link to LICENCE, not code. Just link to the github project page.

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+found at https://github.com/pytorch/pytorch/blob/master/LICENSE
```
Comment (aviks) [inline LICENSE:28]: Better to copy the contents of the LICENSE here.

**Comment by dhairyagandhi96** [comment]
Made the changes
Could you take a look now? Thank you so much!

File: LICENSE (hunk 1 above)
Comment (dhairyagandhi96) [reply LICENSE]: Done

File: LICENSE (hunk 2 above)
Comment (dhairyagandhi96) [reply LICENSE]: Added the contents here

**Pull request merged**